content/app/development/ux/logo/_index.en.md

```diff
@@ -10,7 +11,8 @@ Setting application logo is only available with `Altinn.App.Core` version 7.14.0
 
 {{% /notice%}}
 
-To use organisation logo you can specify that in `applicationmetadata.json`:
+## Use logo from organization
+To use the organization logo you can specify that in `applicationmetadata.json`:
 ```json
 // applicationmetadata.json
```

Comment:
Hvis du vil så kan du bruke `code-title`-shortcode for å vise hvilken fil det gjelder:
```
{{< code-title >}}
App/config/applicationmetadata.json
{{< /code-title >}}
```